nservicebus/rabbitmq/cluster.md

```diff
@@ -3,13 +3,13 @@ title: Using a RabbitMQ cluster
 summary: Guidelines for using RabbitMQ in a cluster
 ---
 
-The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html) recommends not bake in the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster in the client applications and instead use an approach such DNS or load balancing. The same is recommended when using the NServiceBus RabbitMQ transport. For more details see the guide.
+The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html) recommends that client applications should not bake in the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster. An approach such as DNS or load balancing should be used instead. This approach is recommended when using the NServiceBus RabbitMQ transport. For more details, see the guide.
```

Comment:
ok dont worry about it. just figured if it was simple we could link to it.

yep and i see the paragraph starts with the guide